usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -71,11 +71,11 @@ SELECT
 
   UTM.action_date,
   DATE(UTM.action_date + interval '3 months') AS fiscal_action_date,
-  AWD.period_of_performance_start_date,
-  AWD.period_of_performance_current_end_date,
-  FPDS.ordering_period_end_date,
+  UTM.period_of_performance_start_date,
+  UTM.period_of_performance_current_end_date,
+  UTM.ordering_period_end_date,
   UTM.fiscal_year AS transaction_fiscal_year,
-  AWD.fiscal_year AS award_fiscal_year,
+  FY(UTM.award_certified_date) as award_fiscal_year,
```

Comment:
Why not also store `award_fiscal_year` as field in the `UTM` matview, which is derived at matview-refresh time using the `FY(...)` function. That would save on the expense of calculating FY() at view-query-time.

Reply:
This is OBE

Reply:
Done